lib/widgets/static_loader.dart

```diff
@@ -1,11 +1,29 @@
+import 'dart:developer';
+
+import 'package:firebase_remote_config/firebase_remote_config.dart';
 import 'package:flutter/material.dart';
 
 class StaticLoader extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Container(
-        width: 0.0,
-        height: 0.0,
-        decoration: BoxDecoration(color: Color.fromRGBO(94, 94, 94, 1.0)));
+    return Container();
+
+      // Container(
+      //   width: 0.0,
+      //   height: 0.0,
+      //   decoration: BoxDecoration(color: Color.fromRGBO(94, 94, 94, 1.0)));
```

Comment:
You have a lot of commented code in the app, maybe we can remove it a future release